feat: add Console local network guide (#70)
AshAvalanche/ash-docs: docs/console/guides/local-network/node-ids.md

---
sidebar_position: 4
---

# 2. Node ID Secrets Generation

:::caution
The Ash Console is currently in alpha and **not production-ready**. It is under active development and subject to breaking changes.
:::

The Ash Console **decorelates the node ID from Avalanche nodes**, so that you can have a better control over your validator nodes' lifecycle.

> Node IDs are stored as a [secret](/docs/console/glossary#secret) in the Console.

Reviewer: ```suggestion
Node IDs are stored as [secrets](/docs/console/glossary#secret) in the Console.
```

> In this part of the guide, we will create **5 node ID secrets** for the 5 nodes of our local Avalanche network.

Reviewer: ```suggestion
In this part of the guide, we will create **5 node ID secrets** for the 5 nodes of our Avalanche devnet.
```

:::tip
The Node ID secrets need to match the hardcoded Node IDs in the [`genesis_local.json`](https://github.com/ava-labs/avalanchego/blob/master/genesis/genesis_local.json#L47) file of the [AvalancheGo](https://github.com/ava-labs/avalanchego) codebase.
:::

## Fetch the 5 node IDs certificates

> The ["Local Test Network Creation"](/docs/toolkit/ansible-avalanche-collection/tutorials/local-test-network) tutorial uses the same certificates for the 5 nodes of the local network. In this step, we will fetch the certificates files from the [`ansible-avalanche-getting-started`](https://github.com/AshAvalanche/ansible-avalanche-getting-started) repository.

Reviewer: ```suggestion
The [Local Test Network Creation tutorial](/docs/toolkit/ansible-avalanche-collection/tutorials/local-test-network) uses the same certificates and keys for the 5 validator nodes. Let's fetch the certificate and key files from the [`ansible-avalanche-getting-started`](https://github.com/AshAvalanche/ansible-avalanche-getting-started) repository.
```

1. If not already done, create a folder for this guide and navigate to it:

   ```bash

>    mkdir -p ash-console-guides/local-network
>    cd ash-console-guides/local-network

Reviewer: ```suggestion
   mkdir -p ash-console-guides/devnet-network
   cd ash-console-guides/devnet-network
```

   ```

> 2. Fetch the local nodes certificates from the [`ansible-avalanche-getting-started`](https://github.com/AshAvalanche/ansible-avalanche-getting-started/tree/main/files/staking) repository:

Reviewer: ```suggestion
2. Fetch the local nodes certificates and keys from the [`ansible-avalanche-getting-started`](https://github.com/AshAvalanche/ansible-avalanche-getting-started/tree/main/files/staking) repository:
```
